nifi-nar-bundles/nifi-aws-bundle/nifi-aws-processors/src/main/java/org/apache/nifi/processors/aws/cloudwatch/PutCloudWatchMetric.java

```diff
@@ -70,6 +70,33 @@
     public static final Set<Relationship> relationships = Collections.unmodifiableSet(
             new HashSet<>(Arrays.asList(REL_SUCCESS, REL_FAILURE)));
 
+    private static final Set<String> units = Collections.unmodifiableSet(
```

Comment:
Minor, but if you want, you can use ImmutableSet.of(...) from guava

Reply:
Didn't know that. Thanks for the info!

Reply:
10-4